<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormProveedorBase</class>
 <widget class="QWidget" name="FormProveedorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="GBTitulo">
     <property name="title">
      <string>Proveedor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="LRazonSocial">
          <property name="text">
           <string>Razon social:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LERazonSocial"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="LDireccion">
          <property name="text">
           <string>Direccion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="TEDireccion"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="LCuit">
          <property name="text">
           <string>CUIT/CUIL:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ELECuitCuil" name="LECUIT">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="LFax">
          <property name="text">
           <string>Fax:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEFax"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="LTelFijo">
          <property name="text">
           <string>Telefono fijo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETelFijo"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="LTelCel">
          <property name="text">
           <string>Telefono celular:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LETelCel"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="LEmail">
          <property name="text">
           <string>Em@il:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LEEmail"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ELECuitCuil</class>
   <extends>QLineEdit</extends>
   <header location="global">ELECuitCuil.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
